Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -3404,12 +3399,6 @@ void WebPage::mouseEvent(FrameIdentifier frameID, const WebMouseEvent& mouseEven
     m_userActivity.impulse();
```

Comment:
Do we think it's okay to newly elide this call? (We had discussed preserving it earlier)

Reply:
`m_userActivity` seems to just be used for app nap. It doesn’t seem deliberate to consider the unhandled events as user activity. Curious if any tests say otherwise.